src/sentry/features/__init__.py

```diff
@@ -209,6 +209,9 @@
 default_manager.add("projects:plugins", ProjectPluginFeature)
 default_manager.add("users:notification-slack-automatic", UserFeature)
 
+# Workflow 2.0 commit association
+default_manager.add("organizations:commit-event-association", OrganizationFeature)
```

Comment:
Is there a page where features are enabled for orgs?

Reply:
I'm not sure, I just wanted to put something down for now and figure out what an appropriate flag would be later.